core/frontend/src/ModelState.ts

```diff
@@ -113,15 +113,36 @@ export abstract class GeometricModelState extends ModelState implements Geometri
   /** @internal */
   public createTileTreeReference(view: ViewState): TileTreeReference {
     // If this is a reality model, its tile tree is obtained from reality data service URL.
+
     const url = this.jsonProperties.tilesetUrl;
     if (undefined !== url) {
       const spatialModel = this.asSpatialModel;
-      return createRealityTileTreeReference({
-        url,
+
+      if (url.orbitGtBlob === undefined) {

```

Comment:
Up until now a GeometricModel's `tilesetUrl` JSON property was always a string. Existing code will potentially break if it now encounters an object instead of a string. Did you consider adding a different JSON property instead of making the existing `tilesetUrl` property serve dual duty?

Reply:
tilesetUrl was already either a URL to a tileset JSON root document if the point cloud was a TileTree, or an OrbitGtBlobProps object if it's an OPC format point cloud. 

Here's an existing example: https://github.com/imodeljs/imodeljs/blob/master/core/frontend/src/ContextRealityModelState.ts#:~:text=/**%20%40internal%20*/,%7D)%20%3A

We are not keen on it being this way and are discussing using an alternative data structure in future.

Reply:
Thanks, I'm going to review that. I thought the decision to do that was based on some existing code taking that approach, but looking back I'm not able to see it. It would make far more sense to have it separate. We've been discussing simplifying the ways reality data is referenced anyway.

Reply:
I've updated the orbitGtBlob to be a separate property (and not reuse tilsetUrl).
There is a PR for MstnBridge on iModel02 for  that creates the iModel with this alteration.